README.md

```diff
@@ -148,16 +148,18 @@ let target: [ArraySection<Model, String>] = [
 let changeset = StagedChangeset(source: source, target: target)
 ```
 
-You can incremental updates UITableView and UICollectionView using the created `StagedChangeset`.  
-**Don't forget** to update **synchronously** the data referenced by the data-source in `setData` closure, as the differences is applied in stages:
+You can perform incremental updates on `UITableView` and `UICollectionView` using the created `StagedChangeset`.  
+
+⚠️ **Don't forget to *synchronously* update the data referenced by the data-source, with the data passed in the `setData` closure. The differences are applied in stages, and failing to do so is bound to create a crash:**
```

Comment:
Do you think that it's not necessary to be in bold?
It's stand out enough by emoji.

Reply:
I guess that's true. But I just wanted to make it prominent, because I make the same mistake when I started using DifferenceKit. 🤷🏻‍♂️